src/DIG/Tabs.jsx

```diff
@@ -1,4 +1,5 @@
 let { items, variant, size, ...forwardedProps } = props;
+console.log("items", items);
```

Comment:
Do you want to commit these for debug purposes?

Reply:
@charleslavon I did not, the console.log has been removed in the latest commit.